src/Pages/EmailList/EmailList.js

```diff
@@ -0,0 +1,154 @@
+import React, { Component } from 'react';
+import './email-list.css';
+import { getAllUsers } from '../../APIFunctions/User';
+import {
+  Button,
+  ButtonDropdown,
+  DropdownToggle,
+  DropdownMenu,
+  DropdownItem
+} from 'reactstrap';
+
+export default class EmailList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      users: [],
+      filteredUsers: [],
+      filtered: false,
+      dropdownOpen: false
+    };
+  }
+
+  componentDidMount() {
+    if (this.props.user) {
```

Comment:
you can assume `this.props.user` will have a value if you define the component in `signedInRoutes` in `src/Routing.js` [here](https://github.com/SCE-Development/Core-v4/blob/48c9a6340b8001989fe43f0e74cdfa12174fc19c/src/Routing.js#L47)

```suggestion
```

Reply:
added this back, gave a warning in Travis
![image](https://user-images.githubusercontent.com/35363054/82510954-c5bb3480-9ac0-11ea-9b94-c07ab6dc6f93.png)